jsonpatch.go

```diff
@@ -8,20 +8,20 @@ import (
 	"strings"
 )
 
-var errBadJSONDoc = fmt.Errorf("Invalid JSON Document")
+var errBadJSONDoc = fmt.Errorf("invalid JSON Document")
 
-type JsonPatchOperation struct {
+type Operation struct {
```

Comment:
@tamalsaha Why changing the name?
Controller-runtime is considering switching to use `appscode`'s fork.
But it can be a problem for the existing users due to the breaking change.

Reply:
I am not sure what can we do in our fork. Will adding a type alias help?

Reply:
That should help

Reply:
I have added the type alias to 1.0.0 release.